dpnp/dpnp_utils/dpnp_utils_statistics.py

```diff
@@ -81,6 +80,13 @@ def _get_2dmin_array(x, dtype):
         if y is not None:
             dtypes.append(y.dtype)
         dtype = dpt.result_type(*dtypes)
+        # TODO: remove when dpctl.result_type() is fixed
```

Comment:
```suggestion
        dtype = dpnp.result_type(*dtypes)
        # TODO: remove when dpctl.result_type() is returned dtype based on fp64
```

Reply:
It will be fixed in the next PR.
thanks